code/modules/reagents/newchem/medicine.dm

```diff
@@ -0,0 +1,837 @@
+	result_amount = 3
+
+/datum/chemical_reaction/synthflesh
+	name = "Synthflesh"
+	id = "synthflesh"
+	result = "synthflesh"
+	required_reagents = list("blood" = 1, "carbon" = 1, "styptic_powder" = 1)
+	result_amount = 3
+
+/datum/chemical_reaction/styptic_powder
+	name = "Styptic Powder"
+	id = "styptic_powder"
+	result = "styptic_powder"
+	required_reagents = list("aluminium" = 1, "hydrogen" = 1, "oxygen" = 1, "sacid" = 1)
+	result_amount = 4
+	mix_message = "The solution yields an astringent powder."
+
+datum/reagent/omnizine
+	name = "Omnizine"
+	id = "omnizine"
+	description = "Heals 1 of each damage type a cycle. If overdosed it will deal significant amounts of each damage type."
+	reagent_state = LIQUID
+	color = "#C8A5DC"
+	metabolization_rate = 0.2
+	overdose_threshold = 30
+
+datum/reagent/omnizine/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	M.adjustToxLoss(-1*REM)
+	M.adjustOxyLoss(-1*REM)
+	M.adjustBruteLoss(-1*REM)
+	M.adjustFireLoss(-1*REM)
+	..()
+	return
+
+datum/reagent/omnizine/overdose_process(var/mob/living/M as mob)
+	M.adjustToxLoss(3*REM)
+	M.adjustOxyLoss(3*REM)
+	M.adjustBruteLoss(3*REM)
+	M.adjustFireLoss(3*REM)
+	..()
+	return
+
+datum/reagent/calomel
+	name = "Calomel"
+	id = "calomel"
+	description = "Quickly purges the body of all chemicals. If your health is above 20, toxin damage is dealt. When you hit 20 health or lower, the damage will cease."
+	reagent_state = LIQUID
+	color = "#C8A5DC"
+
+datum/reagent/calomel/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	for(var/datum/reagent/R in M.reagents.reagent_list)
+		if(R != src)
+			M.reagents.remove_reagent(R.id,5)
+	if(M.health > 20)
+		M.adjustToxLoss(5*REM)
+	..()
+	return
+
+/datum/chemical_reaction/calomel
+	name = "Calomel"
+	id = "calomel"
+	result = "calomel"
+	required_reagents = list("mercury" = 1, "chlorine" = 1)
+	result_amount = 2
+	required_temp = 374
+
+datum/reagent/potass_iodide
+	name = "Potassium Iodide"
+	id = "potass_iodide"
+	description = "Reduces low radiation damage very effectively."
+	reagent_state = LIQUID
+	color = "#C8A5DC"
+
+datum/reagent/potass_iodide/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	if(M.radiation > 0)
+		if(prob(80))
+			M.radiation--
+	if(M.radiation < 0)
+		M.radiation = 0
+	..()
+	return
+
+/datum/chemical_reaction/potass_iodide
+	name = "Potassium Iodide"
+	id = "potass_iodide"
+	result = "potass_iodide"
+	required_reagents = list("potassium" = 1, "iodine" = 1)
+	result_amount = 2
+
+datum/reagent/pen_acid
+	name = "Pentetic Acid"
+	id = "pen_acid"
+	description = "Reduces massive amounts of radiation and toxin damage while purging other chemicals from the body. Has a chance of dealing brute damage."
+	reagent_state = LIQUID
+	color = "#C8A5DC"
+
+datum/reagent/pen_acid/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	if(M.radiation > 0)
+		M.radiation -= 7
+	M.adjustToxLoss(-4*REM)
```

Comment:
make this a 70% chance to reduce tox loss by 4; closest approximation to 70% of cycles, guaranteed.